Source/Core/CesiumTerrainProvider.js

```diff
@@ -382,7 +415,15 @@ define([
          * @constant
          * @default 2
          */
-        WATER_MASK: 2
+        WATER_MASK: 2,
+        /**
+         * A bounding-volume hierarchy is included as an extension to the tile mesh
+         *
+         * @type {Number}
+         * @constant
+         * @default 3
+         */
+        BVH: 3
```

Comment:
Are we sure this shouldn't be spelled out like `WATER_MASK`?  Perhaps similar question for the extension spec if that is relevant.